lib/rouge/lexers/ghc_cmm.rb

```diff
@@ -245,7 +241,7 @@ class GHCCmm < RegexLexer
         #   `(type /* optional whitespace */ var_name /* optional whitespace */)`
         # Note: Only the token for type is produced here.
         rule %r{
-                (^#{id})
+                (#{id})
```

Comment:
Again, this form was used frequently mid-line, there's no reason it should be anchored. Previously the `(` at the beginning made the `^` a no-op.